whisper.py

```diff
@@ -95,6 +95,9 @@ def _py_fallocate(fd, offset, len_):
 CACHE_HEADERS = False
 AUTOFLUSH = False
 FADVISE_RANDOM = False
+# Buffering setting applied to all operations that do *not* require
+# a full scan of the file in order to minimize cache thrashing.
+BUFFERING = 0
```

Comment:
Well. I'm just not sure that 0 is a good idea for default. OTOH there's no easy way how you can change that  programmatically....